external-crates/move/crates/move-vm-runtime/src/execution/values/values_impl.rs

```diff
@@ -2609,7 +2802,7 @@ fn invariant_violation<S: serde::Serializer>(message: String) -> S::Error {
     )
 }
 
-impl<'a, 'b> serde::Serialize for AnnotatedValue<'a, 'b, MoveTypeLayout, Value> {
+impl serde::Serialize for AnnotatedValue<'_, '_, MoveTypeLayout, Value> {
```

Comment:
This is so _clean_ 

<img src="https://media4.giphy.com/media/NEvPzZ8bd1V4Y/giphy.gif?cid=bd3ea57e7gpuegqobsx3qo187ytica2fhz5v7enach06t7um&ep=v1_gifs_search&rid=giphy.gif&ct=g"/>